src/controllers/user.ts

```diff
@@ -60,4 +60,33 @@ export const UserController = {
 			res.status(500).json({ message: "Internal Server Error" });
 		}
 	},
+
+  updateByDisplayId: async (req: Request, res: Response) => {
+    try {
+      const displayId = req.params.displayId;
+      const updatedData = req.body;
```

Comment:
"更新データ"なので、現在形のほうがより良いと思います

```suggestion
      const updateData = req.body;
```